Makefile

```diff
@@ -405,6 +405,7 @@ GATEWAY_API_RAW_REPO ?= https://raw.githubusercontent.com/$(GATEWAY_API_REPO)
 GATEWAY_API_CRDS_STANDARD_URL = github.com/$(GATEWAY_API_REPO)/config/crd/standard?ref=$(GATEWAY_API_VERSION)
 GATEWAY_API_CRDS_EXPERIMENTAL_URL = github.com/$(GATEWAY_API_REPO)/config/crd/experimental?ref=$(GATEWAY_API_VERSION)
 GATEWAY_API_RAW_REPO_URL = $(GATEWAY_API_RAW_REPO)/$(GATEWAY_API_VERSION)
+KIC_CRDS_URL ?= github.com/kong/kubernetes-ingress-controller/config/crd
```

Comment:
Can we add the version specifier in the same way that we have it for [Gateway API](https://github.com/Kong/gateway-operator/blob/5f0906a3473ff3a114a5e3d3ab2b358c720fafad/pkg/utils/test/zz_generated_gateway_api.go#L1-L12)? So that we use the [KIC version from go.mod](https://github.com/Kong/gateway-operator/blob/90fcea1dbc9244260245e3a9ba770a7bcdc667de/go.mod#L18-L19) when installing CRDs.

Reply:
I've added a `go list ...` directive to fetch KIC's version directly from `go.mod`. PTAL